docs/README.md

```diff
@@ -38,123 +38,20 @@ If you plan to contribute changes to the documentation or website, please submit
 
 MXNet's Python documentation is built with [Sphinx](https://www.sphinx-doc.org) and a variety of plugins including [pandoc](https://pandoc.org/), and [recommonmark](https://github.com/rtfd/recommonmark).
 
-More information on the dependencies can be found in the [CI folder's installation scripts](https://github.com/apache/incubator-mxnet/tree/master/ci/docker/install/ubuntu_docs.sh).
-
-You can run just the Python docs by following the instructions in the Python API guide.
+You can run just the Python docs by following the instructions in the `python_docs/README.md`.
 
 ## Other API Docs
 
 The docs are hosted on the website in each language API's section. You can find installation and build instructions there.
 
 ## How to Build the MXNet Website for Development and QA
 
-`conda` or `miniconda` is recommended.
-* [Conda](https://www.anaconda.com/distribution/#download-section) (install to PATH)
-
-If you only need to make changes to tutorials or other pages that are not generated from one of the API source code folders, then you can use a basic Python pip or conda installation. But if you want edit the API source and have the reference API docs update, you also need to build MXNet from source. Refer to the build from source instructions for this requirement.
-
-
-### Ubuntu Setup
-
-As this is maintained for CI, Ubuntu is recommended. Refer to [ubuntu_doc.sh](https://github.com/apache/incubator-mxnet/tree/master/ci/docker/install/ubuntu_docs.sh) for the latest install script.
+If you only need to make changes to tutorials or other pages that are not generated from one of the API source code folders, then you can install a pre-build MXNet binary. But if you want edit the API source and have the reference API docs update, you also need to build MXNet from source. Refer to the [build from source instructions for this requirement](https://mxnet.apache.org/versions/master/get_started/build_from_source.html).
```

Comment:
one more nitpick: there could be a link for "install a pre-build(t) MXNet binary" too

Reply:
Would https://mxnet.apache.org/versions/master/get_started or pypi be more helpful?

Reply:
The get_started page should work.